### ISSUE TITLE
Update the AStyle extension on Visual Studio Marketplace

### ISSUE THREAD

**Issue by bjornblissing** [lukamicoder/astyle-extension#17]
As of now the current version of this extension on the marketplace is from 2017-04-12. 
I would be nice if the marketplace extension could be upgraded to the latest development. Especially since the latest development include both support for AStyle 3.1 (#11) and Visual Studio 2019 (#14, #16).

**Comment by tomjnaim** [comment]
I agree, I just ran the issue of the marketplace version not including the latest astyle. I am using Atmel studo 7.0 and am having trouble installing it from source, an update on the market place would be appreciated!

**Comment by ghost** [comment]
Latest build 

[AStyleExtension.zip](https://github.com/lukamicoder/astyle-extension/files/3684809/AStyleExtension.zip)

**Comment by bjornblissing** [comment]
@lurumdare I am well aware of this. Still it would be much easier if the marketplace extension would come with the latest version.

**Comment by piyushsoni** [comment]
I'm sorry, can someone please help on how do I use the zip file for VS 2019? Life is difficult without this add-on ...

**Comment by nneesshh** [comment]
Support vs2019 is easy, just unzip the vsix file and modify the extension.vsixmanifest. Increase the version range from "Version="[12.0,15.0)">" to "Version="[12.0,17.0)">"
